hi-ml/src/health_ml/networks/layers/attention_layers.py

```diff
@@ -24,6 +24,21 @@ def forward(self, features: Tensor) -> Tuple[Tensor, Tensor]:
         return (attention_weights, pooled_features)
 
 
+class MaxPoolingLayer(nn.Module):
```

Comment:
Could you add a test for this class in `test_attentionlayers.py`, similar to `test_mean_pooling()`?

Reply:
Done, thanks